aptos-move/block-executor/src/executor.rs

```diff
@@ -538,53 +545,39 @@ where
                         finalized_groups.push((group_key, metadata_op, finalized_group));
                     },
                     Err(err) => {
-                        maybe_code_error = Err(err.into());
-                        break;
+                        return Err(err.into());
```

Comment:
So all error condition here trigger a fallback, right? And therefore we can just return here, so we exit the worker loop and halt the block, falling to sequential?

Reply:
yes, we can forward all errors now.